projects/plugins/jetpack/extensions/blocks/subscriptions/settings.js

```diff
@@ -3,96 +3,232 @@ import { getRedirectUrl } from '@automattic/jetpack-components';
+	return (
+		<a
+			target="_blank"
+			rel="noopener noreferrer"
+			href={ getRedirectUrl( slug ) }
+			className="jetpack-newsletter-link"
+		>
+			{ children }
+		</a>
+	);
+}
+
+function getReachForAccessLevelKey(
+	accessLevelKey,
+	emailSubscribers,
+	paidSubscribers,
+	socialFollowers
+) {
 	if ( emailSubscribers === null || paidSubscribers === null || socialFollowers === null ) {
-		return '';
+		return 0;
 	}
 
-	switch ( accessOptions[ key ] ) {
-		case accessOptions.everybody:
-			return '(' + ( emailSubscribers + socialFollowers ) + '+)';
-		case accessOptions.subscribers:
-			return '(' + emailSubscribers + ')';
-		case accessOptions.paid_subscribers:
-			return '(' + paidSubscribers + ')';
+	switch ( accessOptions[ accessLevelKey ].key ) {
+		case accessOptions.everybody.key:
+			return emailSubscribers;
+		case accessOptions.subscribers.key:
+			return emailSubscribers;
+		case accessOptions.paid_subscribers.key:
+			return paidSubscribers;
 		default:
+			return 0;
 	}
 }
 
-export function MisconfigurationWarning( { accessLevel } ) {
+function NewsletterLearnMore() {
 	return (
-		<div className="jetpack-subscribe-notice-misconfiguration warning">
-			{ sprintf(
-				/* translators: %1$s: visibility label for the newsletter, %2$s: label for setting "everybody". this is a warning in the newsletter when posts have a private or password-protected visibility */
+		<small className="jetpack-newsletter-learn-more">
+			{ createInterpolateElement(
 				__(
-					'Private or password-protected posts cannot be assigned a newsletter setting of "%1$s". Please update the setting to "%2$s", or update the post visibility setting.',
+					'Restrict your post to subscribers. <learnMoreLink>Learn more.</learnMoreLink>',
 					'jetpack'
 				),
-				accessOptions[ accessLevel ].label,
-				accessOptions.everybody.label
+				{
+					learnMoreLink: (
+						<Link
+							slug={ getRedirectUrl( 'paid-newsletter-info', {
+								anchor: 'memberships-and-subscriptions',
+							} ) }
+						/>
+					),
+				}
 			) }
-		</div>
+		</small>
 	);
 }
 
-function NewsletterAccessChoices( {
-	accessLevel,
+function NewsletterNotice( { accessLevel, socialFollowers, emailSubscribers, paidSubscribers } ) {
+	// Get the reach count for the access level
+	let reachCount = getReachForAccessLevelKey(
+		accessLevel,
+		emailSubscribers,
+		paidSubscribers,
+		socialFollowers
+	);
+
+	if ( 0 === reachCount ) {
+		return (
+			<FlexBlock>
+				<Notice status="info" isDismissible={ false } className="edit-post-post-visibility__notice">
+					{ createInterpolateElement(
+						__(
+							'You don’t have any subscribers yet. How about <importingLink>importing</importingLink> some? Or check out <thisGuideLink>this guide</thisGuideLink> on how to grow your audience.',
+							'jetpack'
+						),
+						{
+							importingLink: (
+								<Link
+									slug={
+										'https://wordpress.com/support/launch-a-newsletter/import-subscribers-to-a-newsletter/'
```

Comment:
We should localize the link. There's a function for it but can't remember now how to do that in Jetpack.

Reply:
This is done